blog/announcements.xml

```diff
@@ -2,6 +2,115 @@
+                Notable Changes</a></li>
+          </ul>
+        </li>
+        <li><a href="/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="/download.html#download-nixos">Image downloads</a></li>
+      </ul>
+
+      <p>
+        The 21.11 release was possible due to the efforts of <strong>1541</strong>
+        contributors in <strong>41960</strong> commits. We would especially
+        like to thank our top 10 contributors:
+        <a href="https://github.com/supersandro2000">Sandro Jäckel</a>,
+        <a href="https://github.com/fabaff">Fabian Affolter</a>,
+        <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        <a href="https://github.com/figsoda">figsoda</a>,
+        <a href="https://github.com/Artturin">Artturin</a>,
+        <a href="https://github.com/marsam">Mario Rodas</a>,
+        <a href="https://github.com/bobby285271">Bobby Rong</a>,
+        <a href="https://github.com/mic92">Jörg Thalheim</a>,
+        <a href="https://github.com/dotlambda">Robert Schütz</a>,
+        <a href="https://github.com/primeos">Michael Weiss</a>.
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
+      </p>
+
+      <ul>
+        <li>XXX 12985 packages were added</li>
+        <li>XXX 14109 packages were removed</li>
+        <li>XXX 16768 packages were updated</li>
+      </ul>
+
+      <p>
+        Likewise, our NixOS module system got bigger and better:
+      </p>
+
+      <ul>
+        <li>XXX 1527 options were added</li>
+        <li>XXX 290 options removed</li>
+        <li>XXX 400 options were updated</li>
+      </ul>
+
+      <h3>Stabilization Contributors</h3>
+
+      <p>
+        Stabilization of the NixOS happens a month before the planned release. The
+        goal is to have as little as possible continuous integration (Hydra)
+        jobs failing before the release is cut.
+      </p>
+
+      <p>
+        Individuals who contributed to stabilizing this release:
+        <a href="https://github.com/fabaff">Fabian Affolter</a>,
+        <a href="https://github.com/supersandro2000">Sandro Jäckel</a>,
+        <a href="https://github.com/figsoda">figsoda</a>,
+        <a href="https://github.com/trofi">Sergei Trofimovich</a>,
+        <a href="https://github.com/Artturin">Artturin</a>,
+        <a href="https://github.com/alyssais">Alyssa Ross</a>,
+        <a href="https://github.com/thiagokokada">Thiago Kenji Okada</a>,
+        <a href="https://github.com/sternenseemann">Lukas Epple</a>,
+        <a href="https://github.com/TredwellGit">Tredwell</a>,
+        <a href="https://github.com/lovesegfault">Bernardo Meurer</a>, and 477 others!
+      </p>
```

Comment:
Unclear if this needs to be changed (haven't verified with the previous release)

Reply:
I was in a meeting with @tomberek last night when he was filling this part out so these should be up to date and current.